packages/affiliates/test/datasets/set1/coingecko/0xD16c79c8A39D44B2F3eB45D2019cd6A42B03E2A9.json

```diff
@@ -0,0 +1 @@
991],[1602857428914,625038.6407487926],[1602871904277,603774.3175774008],[1602879053673,598228.6965621697],[1602886210188,610930.0785708086],[1602893518104,174462.53668041178],[1602918662867,161891.54081334276],[1602936682124,157481.15170330752],[1602943871674,31093.253639768143],[1602965453851,20816.665748503732],[1602972684473,30424.161154664227],[1602976022671,50614.84545914222],[1602994138231,59767.574325394286],[1602997740015,61057.430824133895],[1603030159144,50932.266339345726],[1603062569352,47197.17903788176],[1603069685481,56914.433896828414],[1603073286813,57544.61261418971],[1603080494996,48127.42774624788],[1603084092982,47323.672794132915],[1603091302249,78670.95352153022],[1603148791825,41737.280787406504],[1603155968005,33321.592815979544],[1603159614955,33151.361047651466],[1603177687270,2122.916246283656],[1603195637025,12415.737398118155],[1603242443862,10405.120769236104],[1603300043975,5477.672317169451],[1603314409896,19881.20129846866],[1603336059087,27302.717617336846],[1603354064076,47557.80271029247],[1603361239858,67274.21315434971],[1603364831751,128128.33641426903],[1603368526940,136281.301463591],[1603372070083,158742.95544051912],[1603379263651,196025.2889212883],[1603386438494,265842.739527999],[1603390048439,321759.6640034919],[1603393632041,375480.8063222384],[1603397263792,425826.35043396195],[1603400853464,504678.18921197805],[1603404455504,510136.4612992637],[1603415282989,524617.0815431112],[1603422417538,527627.8447563495],[1603440573420,500644.0263841348],[1603447728254,481764.36957879824],[1603451349379,424354.90109787317],[1603454963587,514091.5700017431],[1603458545975,494510.2244516759],[1603462167674,473466.81326891115],[1603465733884,457433.0144829754],[1603472990078,413133.84900724405],[1603476549808,354460.8127921246],[1603480157672,322950.39946019294],[1603483720340,289203.194177159],[1603487353012,235956.58121662316],[1603490937144,251706.50953430665],[1603494529106,252459.9175120071],[1603498126928,252359.7088144343],[1603501747677,251947.1057330107],[1603508986639,228034.0120234963],[1603541382626,149552.41229016907],[1603544927805,142414.92926652124],[1603548620034,136986.00243286495],[1603552163553,138892.16628336388],[1603555852546,140652.10029913473],[1603566611703,101757.51385778152],[1603570255176,90924.264533003],[1603573778282,53811.604751088234],[1603577402592,43728.24920646366],[1603627721507,26912.25276502478],[1603638514061,51867.812173603605],[1603642079519,50521.641714884325],[1603656558078,35992.37654126067],[1603663770265,29373.88620814514],[1603674538479,35052.32658536233],[1603692537422,37285.833956231065],[1603724832167,9892.877684292636],[1603750048556,7844.006710592691],[1603760835078,24557.534609230992],[1603775274801,24972.79215474545],[1603778844274,86714.98180440202],[1603786004198,123414.57454102328],[1603789673602,138755.4304582171],[1603796815350,208808.773741179],[1603814840504,208127.1720081098],[1603822064119,235508.07493510685],[1603820394000,235508.07493510685]]}
```

Comment:
Leaving this comment here instead of on the file, since I can't comment on it due to its size.

I think storing small sample datasets in the repo for the purpose of testing is a solid strategy (preferable to mocking responses IMO), but is it necessary to store a 1MM+ line file? Can we cut this by 99.9% and get the same result (1000 lines)? The file is also 18MB -- github has a hard file size limit (i.e. they won't let you push) at 100MB.

I also think cutting it helps with maintenance, too. A file that big can't really be opened in an editor or commented on in GH, so you're forced to do weird stuff, like piping it around to `head`, `tail`, and `grep` just to look at it. It seems like it would make the test a lot easier to maintain if it were ~1000 lines, so it's easy to open in whatever editor to see what's inside. 

All that being said, if it's absolutely necessary, I think it's fine, but it just seems like storing generated files of that size for the purpose of testing may be a little overkill.

Reply:
its a good point, its not absolutely necessary. This is dependent on our datasets time frame which we can cut down significantly! Ill see what i can do here.